src/main/java/fi/helsinki/cs/tmc/core/commands/SendFeedback.java

```diff
@@ -6,6 +6,7 @@
 
 import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
+import java.net.URI;
 
 import java.util.Map;
```

Comment:
These import issues are from the automatic import of NetBeans. Is there a way to configure it so that it would automatically add the imports to their correct locations? It seems tedious having to do this by hand for each file.